lang/en.json

```
@@ -64,9 +64,11 @@
   "DCC.AsCoins": "As Coins",
   "DCC.Attack": "Attack",
   "DCC.AttackBonus": "Attack Bonus",
+  "DCC.AttackBonusRollInvalidFormulaInline": "<b>Invalid Attack Bonus formula '{formula}'</b>",
   "DCC.AttackPl": "Attacks",
   "DCC.AttackRoll": "Attack Roll with {weapon}",
   "DCC.AttackRollEmote": "Attacks with their {weaponName} and hits AC {rollHTML} for {damageRollHTML} points of damage!{crit}{fumble}",
+  "DCC.AttackRollEmoteDeed": "Attacks with their {weaponName} and hits AC {rollHTML} with deed die of {attackBonus} for {damageRollHTML} points of damage!{crit}{fumble}",
```

```suggestion
  "DCC.AttackRollEmoteDeed": "Attacks with their {weaponName} and hits AC {rollHTML} with a deed roll of {attackBonus} for {damageRollHTML} points of damage!{crit}{fumble}",
```

Super minor, but I think "deed die" generally refers to the actual die (i.e. d3, d5, etc) rather than the value of the roll.